backend/run/copr-repo

```diff
@@ -127,8 +127,22 @@ def filter_existing(opts, subdirs):
     return new_subdirs
 
 
+def _database_option(chroot: str) -> str:
```

Comment:
I don't think it makes much sense to keep dead code here.  Would you mind moving the `system_that_needs_db` into config.py (say `CHROOTS_USING_SQLITE_REPODATA`), documenting it, keeping that dict empty by default, and keeping this method used?